### PR TITLE
finish personal portfolio - Add Portfolio and skills

### PR DESCRIPTION
**Pull request by AsciencioAlex**
Creation of about me page
Addition of skills and portfolio page
Addition of Resume
Add of Frameworks and Languages section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
...
+							<h4 class="port-name color-dark">Tonic</h4>
+							<div class="port-description flex-row align-center"></div>
+							<h5 class="port-invent color-info">Canopy</h5>
+							<ul class="job_done flex-row">
+								<li class="items">Back End Dev</li>
+								<li class="items">2015</li>
+							</ul>
+							
+						</div>
+						<p class="description-main">A daily selection of privately personalized reads; no accounts
```
Comment (Bantealem) [inline index.html]: Great job. Please make sure your design is the same as Figma design.
Your design

image

Figma design

image

File: index.html [hunk 2]
```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
...
+								/>
+								HTML
+							</li>
+							<li class="about_page__item">
+								<img
+									src="img/about_page/css.png"
+									alt="javascript png image"
+								/>
+								CSS
+							</li>
```
Comment (Bantealem) [inline index.html:289]: Please make sure your design is the same as Figma design.
Your design

image
 Figma design

image

File: index.html [hunk 3]
```diff
@@ -71,6 +71,250 @@ <h1 class="headline__title color-dark">
...
+				<div class="about flex-column justify-center">
+					<h2 class="about__title color-dark">About Myself</h2>
+					<p class="color-info">
+						Hello I'm a software developer! I can help you build a product,
+						feature or website. Look through some of my work and experience!
+						If you like what you see and have a project you need coded,
+						don't hesitate to contact me.
+					</p>
+					<a href="#">Let's Connect</a>
+					<ul class="social__icons flex-row align-center">
```
Comment (Bantealem) [inline index.html]: Please make sure you used the correct font-color from the FGima design
Your design
image
Figma design
image

**Review by Bantealem — CHANGES_REQUESTED** [review]
Hi @AsciencioAlex , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [x] Correct Github flow
- [x] No Linters Error
- [x] Professional README 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @AsciencioAlex  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**